platforms/posix/cmake/sitl_target.cmake

```diff
@@ -61,7 +61,7 @@ ExternalProject_Add(mavsdk_tests
 set(viewers none jmavsim gazebo)
 set(debuggers none ide gdb lldb ddd valgrind callgrind)
 set(models none shell
-	if750a iris iris_opt_flow iris_opt_flow_mockup iris_vision iris_rplidar iris_irlock iris_obs_avoid iris_rtps solo typhoon_h480
+	if750a iris iris_opt_flow iris_opt_flow_mockup iris_vision iris_rplidar iris_irlock iris_obs_avoid iris_rtps solo typhoon_h480 px4vision
```

Comment:
Please make sure that we keep the alphabetical order here.

Reply:
Done!